example/src/pie/PieChartBasic.js

```diff
@@ -63,12 +64,33 @@ class PieChartBasic extends Component {
     }
 
     return (
-      <View>
-        <Pie
-          data={data}
+      <ScrollView horizontal={true} style={{flex:1,backgroundColor:'#F5FCFF'}} contentContainerStyle={{justifyContent:'center',alignItems:'center'}}>
```

Comment:
I'd say it would probably make best sense as a specific scroll demo chart for a column bar chart... pie chart would not be as likely to scroll horizontally. :)